### PR TITLE
Fix several CVEs in AIX explainer image

### PR DESCRIPTION
**Pull request by andresmascl**
Signed-off-by: andresmascl <andresma@gmail.com>

Fixing critical CVEs in latest kserve/art-explainer version
Comparing this to latest stable `kserve/aix-explainer` image:
grype-tests

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Not sure there's an open issue for this

**Type of changes**
Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

**Feature/Issue validation/testing**:

Please describe the tests that you ran to verify your changes and relevent result summary. Provide instructions so it can be reproduced.
`kserve/aix-explainer` image builds ok with no issues
`Also test_v1beta1_aix_explainer_spec.py` passed
test-new-img- -show-tensorflor-version

- Logs

**Special notes for your reviewer**:

This PR only changes image versions

**Release note**:

```release-note
Fix several critical vulnerabilities (CVEs) in kserve/aix-explainer
```

### CODE REVIEW

**Comment by yuzisun** [comment]
@andresmascl Can you help rebase from the master?

**Comment by andresmascl** [comment]
@yuzisun All done 👍

**Comment by yuzisun** [comment]
> @yuzisun All done 👍

Thanks! looks like there are dependencies issues for python 3.9
```
The conflict is caused by:
    aix360 0.2.1 depends on tensorflow==1.14
    aix360 0.2.0 depends on tensorflow==1.14
```

File: python/aixexplainer/setup.py [hunk 1]
```diff
@@ -33,12 +33,12 @@
...
                  Not intended for use outside KServe Frameworks Images',
     long_description=open('README.md').read(),
-    python_requires='>3.7',
+    python_requires='>=3.9',
     packages=find_packages("aixserver"),
     install_requires=[
         f"kserve>={version}",
         "argparse >= 1.4.0",
-        "aix360 >= 0.2.0",
+        "aix360 >= 0.2.1",
```
Comment (yuzisun) [inline python/aixexplainer/setup.py:41]: seems like 0.2.1 still depends on tensorflow 1.14

**Comment by yuzisun** [comment]
see https://github.com/kserve/kserve/issues/2823, KServe is going to sunset AIX 360 explainer so this PR is no longer needed

**Comment by andresmascl** [comment]
Thank you for letting me know @yuzisun !

**Pull request closed without merging**